pages/common.py

```diff
@@ -6,23 +6,97 @@
 import re
 import settings
 
-# Python 2 compatibility
-def nop2(x,y):
-    return x
-try:
-    unicode = unicode
-except:
-    unicode = nop2
-
-# This function will pass unicode text right through, other text will be decoded as utf-8
-def force_unicode(text):
-    if repr(text)[0] == 'u': #FIXME...
-        return text
-    return unicode(text.strip(),"utf-8")
+site_brand = "Freenet"
+
+# Wrapper for HTML intended to catch markdown-of-html bugs
+# The HTML class taints all strings that are added to it to be HTML as well
+class HTML(object):
+    def __init__(self, text):
+        if not isinstance(text, str):
+            raise TypeError("HTML text must be of str, {f} found".format(f=text.__class__.__name__))
+        self.text = text
+    
+    def __add__(self, other):
+        if isinstance(other, HTML):
+            return HTML(self.text + other.text)
+        return HTML(self.text + other)
+    
+    def __radd__(self, other):
+        if isinstance(other, HTML):
+            return HTML(other.text + self.text)
+        return HTML(other + self.text)
+    
+    def __repr__(self):
+        return u"HTML: " + self.text
```

Comment:
This isn't enclosed in angle brackets, so it should probably be something that will produce the same thing when passed to `eval()`. [0] Thoughts on making this something like `"HTML({})".format(repr(self.text)`?

Is the leading `u` necessary? I thought that was a Python 2 thing for Unicode string literals.

[0] https://docs.python.org/3.5/library/functions.html#repr